packages/web-components/fast-element/src/testing/fixture.spec.ts

```diff
@@ -1,12 +1,9 @@
 import { expect } from "chai";
-import {
-    attr,
-    customElement,
-    FASTElement,
-    html,
-    observable,
-    Updates,
-} from "@microsoft/fast-element";
+import { attr } from "../components/attributes.js";
+import { customElement, FASTElement } from "../components/fast-element.js";
+import { observable } from "../observation/observable.js";
+import { Updates } from "../observation/update-queue.js";
```

Comment:
Should `Updates` be lowercase

Reply:
This is actually following our pattern of making the global const singleton core services uppercased. Other examples include `DOM`, `Markup`, `Parser`, `Compiler`, etc.